src/components/Input/test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { Input, InputUI } from '.';
+
+describe('Input', () => {
+    test('renders', () => {
+        render(<Input name="input" defaultValue="test" placeholder="example" />);
+        const inputEle = screen.getByPlaceholderText('example');
+        fireEvent.change(inputEle, { target: { value: 'changed' } });
+        expect(screen.getByPlaceholderText('example')).toHaveValue('changed');
```

Comment:
This confirms that the input element value changed which is great, but it should probably also confirm that the passed in `onChange` handler gets fired with the right value.